lib/nat/service.go

```diff
@@ -314,7 +314,16 @@ func (s *Service) acquireNewLocked(ctx context.Context, mapping *Mapping, nats m
 func (s *Service) tryNATDevice(ctx context.Context, natd Device, intPort, extPort int, leaseTime time.Duration) (Address, error) {
```

Comment:
Probably bikeshedding but could we switch to a different term? NAT only really makes sense for IPv4.

Reply:
e.g `tryOpenPort` ?

Reply:
Well the whole package is named `nat`, so I think this would lead to much noise or inconsistency...